LanguageExt.Core/Monads/Alternative Value Monads/Either/Either-Shared/EitherData.Extensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
```

Comment:
Could you add `#nullable enable` to the top of the file pls.  language-ext is being slowly migrated to support nullable references, it'd be good if new code supported it off the bat.